uk_election_ids/election_ids.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, election_type, date):
 
         Args:
             election_type (str): May be one of
-                ``['europarl', 'gla', 'local', 'mayor', 'naw', 'nia', 'parl', 'pcc', 'sp', 'senedd']``
+                ``['europarl', 'gla', 'local', 'mayor', 'naw', 'nia', 'parl', 'pcc', 'sp', 'senedd', 'ref']``
```

Comment:
This documentation change needs deploying to the [docs site](https://democracyclub.github.io/uk-election-ids/)
The command to do it is noted in https://github.com/DemocracyClub/uk-election-ids/wiki